File: tello.py

import socket
import time

host = ''
port = 90000
locaddr = (host,port)
tello_address = ('192.168.10.1', 8889)
sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
def send_command(msg):
    msg = msg.encode(encoding="utf-8")
    sent = sock.sendto(bytes(msg), tello_address)


if __name__ == '__main__':
    send_command("command")
    time.sleep(1)
    send_command("takeoff")
    time.sleep(10)
    send_command("up 100")
    time.sleep(5)
    send_command("flip l")
    time.sleep(5)
    send_command("land")

File: lua/tello.lua
local socket = require("socket")
local udp = assert(socket.udp())
local wait_time = 10
local host, port = "192.168.10.1", 8889

udp:sendto("command", host, port)
socket.sleep(wait_time)
udp:sendto("takeoff", host, port)
socket.sleep(wait_time)
udp:sendto("land", host, port)






